ote/src/cljs/ote/views/admin/detected_changes.cljs

```diff
@@ -235,25 +275,28 @@
                          :button-label       "Lataa"
                          :type               :file-and-delete
                          :allowed-file-types [".zip"]
-                         :on-change          #(e! (admin-controller/->UploadAttachment (.-target %)))}]})]
+                         :on-change          #(e! (admin-transit-changes/->UploadAttachment (.-target %)))}]})]
      (get-in app-state [:admin :transit-changes :upload-gtfs])]])
 
 (defn configure-detected-changes [e! app-state]
-  (let [page (:page app-state)
-        tabs [{:label "Tunnista muutokset" :value "admin-detect-changes"}
+  (let [tabs [{:label "Tunnista muutokset" :value "admin-detected-changes"}
               {:label "Reitin tunnistus" :value "admin-route-id"}
               {:label "Lataa gtfs" :value "admin-upload-gtfs"}
               {:label "Sopimusliikenne" :value "admin-commercial-services"}]
-        selected-tab (or (get-in app-state [:admin :transit-changes :tab]) "admin-detect-changes")]
+        selected-tab (or (get-in app-state [:admin :transit-changes :tab]) "admin-detected-changes")
+        recalc? (if (get-in app-state [:admin :transit-changes :hash-recalculations]) true false)]
```

Comment:
`(some? (get-in app-state [:admin :transit-changes :hash-recalculations]))`

Reply:
Better. Changed.